src/app/containers/Video/__snapshots__/index.test.jsx.snap

```diff
@@ -1,16 +1,293 @@
+                    Object {
+                      "model": Object {
+                        "copyrightHolder": "BBC",
+                        "height": 1080,
+                        "locator": "ichef.test.bbci.co.uk/images/ic/$recipe/p01k6mtv.jpg",
+                        "originCode": null,
+                        "width": 1920,
+                      },
+                      "type": "rawImage",
+                    },
+                    Object {
+                      "model": Object {
+                        "blocks": Array [
+                          Object {
+                            "model": Object {
+                              "blocks": Array [
+                                Object {
+                                  "model": Object {
+                                    "blocks": Array [
+                                      Object {
+                                        "model": Object {
+                                          "attributes": Array [],
+                                          "text": "Ants",
+                                        },
+                                        "type": "fragment",
+                                      },
+                                    ],
+                                    "text": "Ants",
+                                  },
+                                  "type": "paragraph",
+                                },
+                              ],
+                            },
+                            "type": "text",
+                          },
+                        ],
+                      },
+                      "type": "altText",
+                    },
+                  ],
+                },
+                "type": "image",
+              },
+            ],
+          },
+          "type": "aresMedia",
+        },
+      ]
+    }
+  />
+</Context.Provider>
+`;
+
+exports[`Video with data but no image should only render the video 1`] = `
+<ForwardRef>
+  <Video
+    duration={299}
+    imageLocator="/cpsprodpb/5BD5/production/_101690532_2.jpg"
+    kind="clip"
+    rawImageSrc="raw image source"
+    versionID="p064nsz3"
+    videoLocator="urn:bbc:pips:pid:p064nsyw"
+  />
+</ForwardRef>
+`;
 
 exports[`Video with data should render the important props in divs 1`] = `
-<Video
-  duration="299"
-  imageLocator="/cpsprodpb/5BD5/production/_101690532_2.jpg"
-  kind="clip"
-  rawImageSrc="https://ichef.bbci.co.uk/news/640/cpsprodpb/5BD5/production/_101690532_2.jpg"
-  versionID="p064nsz3"
-  videoLocator="urn:bbc:pips:pid:p064nsyw"
-/>
+<ForwardRef>
+  <Video
+    duration={299}
+    imageLocator="/cpsprodpb/5BD5/production/_101690532_2.jpg"
+    kind="clip"
+    rawImageSrc="raw image source"
+    versionID="p064nsz3"
+    videoLocator="urn:bbc:pips:pid:p064nsyw"
+  />
+</ForwardRef>
 `;
 
-exports[`Video with no data should not render anything 1`] = `null`;
+exports[`Video with no data should not render anything 1`] = `
```

Comment:
We should ensure that the VideoContainer with no `aresMedia` block should render `null`.  I know that we discussed that we weren't going to use values within `aresMedia` block within this PR, but we can just simply return null if this block does not exist.